examples/client_example/README.rst

```diff
@@ -0,0 +1,92 @@
+Python Client Example
+#####################
+
+Introduction
+============
+
+Python Client Example, using ``python-tuf``.
+
+For information about installing ``python-tuf``, please refer to the
+`Installation documentation <https://theupdateframework.readthedocs.io/en/latest/INSTALLATION.html>`_.
+
```

Comment:
Any reason to not use Markdown?

Reply:
No reason to use Markdown, actually I did start with Markdown, but I thought it could be used with the RTD/Sphinx.

Reply:
RTD can do Markdown too.

Reply:
Should we agree to move to Markdown?